### DISCUSSION TITLE
Export starter game boot

### DISCUSSION THREAD

**Discussion by lurgx** [godotengine/godot-proposals#4380] [Platforms]
It would be great, although I know that it is something very complex, at least only to export, I suppose it will be somewhat laborious to be able to export directly to the operating system.
More like a distribution as if it were played in an arcade
I mean to put a usb to the cpu and be able to play a game made in godot without needing to have an operating system
And I tried this using linux making a distribution and that it starts a godot game without having to open it, so when it starts it will seem that you are starting an arcade and not a computer
Examples would be like the LAKKA or Batocera emulator, it would be good to be able to do this, this would allow players to enter the Arcade market, such as Scaffolding, Konami Taito, etc.
using a phyton with arduino and godot my imagination has run wild and it seems that it would work if a game could be exported as a usb boot
I hope you can understand me

**Comment by Calinou** [comment]
This is likely better suited as a third-party effort, as Godot's goal is to develop a game engine, not an operating system :slightly_smiling_face:
